artacle-swagger.yaml

```diff
@@ -918,6 +1194,202 @@ components:
               nullable: true
               example: "0x9df947cd8a9810964473339c38a95201ecbe112d"
 
+    CollectionTraits:
+      type: object
+      required:
+        - traitValue
+        - valuationAllTime
+        - valuationAllTimeUSD
+        - valuationRecent
+        - valuationRecentUSD
+        - recentPeriodMonths
+        - totalAmountOfSales
+        - medianTTFAllTime
+        - medianTTFRecentPeriod
+        - quantity
+        - largeDistinctCount
+      properties:
+        traitValue:
+          type: string
+          description: Trait value in form of trait_name:value
+          example: Color:Red
+        valuationAllTime:
+          description: Trait valuation for all time
+          type: number
+          format: float
+        valuationAllTimeUSD:
+          description: Trait valuation for all time
+          type: number
+          format: float
+        valuationRecent:
+          description: Trait valuation for recent period
+          type: number
+          format: float
+        valuationRecentUSD:
+          description: Trait valuation for recent period
+          type: number
+          format: float
+        recentPeriodMonths:
+          description: Recent period of trait valuation
+          type: number
+          format: uint32
+        totalAmountOfSales:
+          description: Amount of sales
+          type: number
+          format: uint32
+        medianTTFAllTime:
+          description: Median trait to floor for all time
+          type: number
+          format: float
+        medianTTFRecentPeriod:
+          description: Median trait to floor for recent period
+          type: number
+          format: float
+        tokensCountWithTrait:
+          description: Count of tokens with this trait
+          type: number
+          format: uint32
+          example: 1
+        largeDistinctCount:
+          description: Indicates if trait value is insignificant
+          type: boolean
+        floor:
+          description: Trait floor
+          type: number
+          format: float
+        floorUSD:
+          description: Trait floor
+          type: number
+          format: float
+        totalSalesVolume:
+          description: Total sales volume
+          type: number
+          format: float
+        totalSalesVolumeUSD:
+          description: Total sales volume
+          type: number
+          format: float
+                
+    CollectionTrait:
```

Comment:
What is the difference with CollectionTraits? It looks like only CollectionTraiT must be kept and referred to in the methods